apps/events/viewsets/invitation.py

```diff
@@ -0,0 +1,84 @@
+from drf_yasg.utils import swagger_auto_schema, no_body
+from rest_framework import viewsets, status
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from apps.events.models import Invitation, Event
+from apps.events.serializers import InvitationSerializer
+from apps.notifications.handlers.in_app import InAppNotificationsHandler
+from apps.notifications.managers.chain import NotificationsChainManager
+from apps.notifications.models import Notification
+from apps.profiles.utils import get_user_object, is_current_user
+
+
+class InvitationViewSet(viewsets.ModelViewSet):
+    model = Invitation
+    permission_classes = IsAuthenticated
+    queryset = Invitation.objects.all()
+    serializer_class = InvitationSerializer
+    http_method_names = ["get", "post", "delete"]
+
+    handlers = (InAppNotificationsHandler(),)
+    notifications_manager = NotificationsChainManager(handlers)
+
+    @swagger_auto_schema(request_body=no_body)
+    @action(methods=["post"], detail=True, url_name="invite_accept")
+    def invite_accept(self, request, event_id):
+        event = Event.objects.filter(event_id=event_id).first()
+        user = get_user_object(request.user.id)
+        is_current_user(request, user)
+
+        invitation_object = Invitation.objects.filter(event=event, recipient=user).first()
```

Comment:
same goes here

Reply:
`invitation_object` can be just `invitation`, don't see any conflicts here with Global scope :)

Reply:
check that event_id from invite is the same with event id from request

Reply:
check user permissions to work with this invitation ( recipient should be current user)

Reply:
if user has access to invite and he is recipient, we can add him to event paticipants